### ISSUE TITLE
Another Suggestion (visible area)

### ISSUE THREAD

**Issue by OliviaOliveiira** [kijai/ComfyUI-LuminaWrapper#20]
Hey there, once again!
I was wondering if you could implement the same logic to see the actual area the prompt will correspond to as it was done here, a loooong time ago 
https://github.com/Davemane42/ComfyUI_Dave_CustomNode

**Comment by kijai** [comment]
Yeah maybe, it's very simple though as it only supports dividing the area evenly. I tried manually set custom areas and masks, but it doesn't seem to work.
